### PR TITLE
kraken: show_codes = true for all APIs

### PR DESCRIPTION
**Pull request by azime**
Jira: http://jira.canaltp.fr/browse/NAVP-505

### CODE REVIEW

**Comment by azime** [comment]
jenkins test this

**Comment by azime** [comment]
Jenkins, test this please

File: source/jormungandr/tests/routing_tests.py [hunk 1]
```diff
@@ -407,6 +407,16 @@ def test_novalidjourney_on_first_call_debug(self):
         eq_(response['journeys'][1]['arrival_date_time'], "20120614T160000")
         eq_(response['journeys'][1]['type'], "best")
 
+    def test_codes_on_object(self):
```
Comment (antoine-de) [inline source/jormungandr/tests/routing_tests.py]: can you add a comment on the method ?

And maybe add a method that do the same test but with `show_codes=false`

Comment (azime) [inline source/jormungandr/tests/routing_tests.py]: OK

**Comment by azime** [comment]
corrections

**Pull request merged**